internal/pkg/source/source.go

```diff
@@ -21,14 +21,14 @@ import (
 	"strings"
 
 	"github.com/eapache/queue"
-	"github.com/google/go-flow-levee/internal/pkg/config"
 	"github.com/google/go-flow-levee/internal/pkg/sanitizer"
 	"github.com/google/go-flow-levee/internal/pkg/utils"
 	"golang.org/x/tools/go/analysis/passes/buildssa"
 	"golang.org/x/tools/go/ssa"
 )
 
 type classifier interface {
+	IsSource(types.Type) bool
 	IsSanitizer(*ssa.Call) bool
 	IsPropagator(*ssa.Call) bool
 	IsSourceFieldAddr(*ssa.FieldAddr) bool
```

Comment:
Doesn't need to be part of this PR, but `type classifier interface` should likely belong to either its own package or to `config`.  We'll want to reuse this interface in other analyzers as those are extracted.

Reply:
Perhaps.
There is a school of thought that suggests to define interfaces where they are used. Therefore, according to this particular approach the interfaces is placed properly.
The reason for defining this interface is that config has too many responsibilities - a lot of which are not interesting to the Source package.